UnityAds/UnityAdsAdapterInitializationDelegate.h

```diff
@@ -0,0 +1,32 @@
+//
+//  UnityAdsAdapterInitializationDelegate.h
+//  MoPubSDK
+//
+//  Copyright © 2021 MoPub. All rights reserved.
+//
+
+#ifndef UnityAdsAdapterInitializationDelegate_h
+#define UnityAdsAdapterInitializationDelegate_h
+
+#import <UnityAds/UnityAds.h>
+
+/**
+ TODO: Question: Why is it necissary to wrap UnityAdsInitializationDelegate like this?
+ TODO: doc string.
+ */
+@interface UnityAdsAdapterInitializationDelegate : NSObject<UnityAdsInitializationDelegate>
```

Comment:
I would probably name it something like `UnityAdsInitializationDelegateAdapter`. To my mind this is the closest purpose of the class. In this case we wont need any docs required since it should be pretty much explanatory.

The purpose of the class basically to convert our interface to `their` interface.

Reply:
👍 This name was just copy and pasted from the existing adapter.  I didn't really like it too much (and honestly I was hoping that maybe we could remove the need for the adapter, but I wasn't sure, so I just left the name as it was)